improver/cli/__init__.py

```diff
@@ -207,11 +207,12 @@ def create_constrained_inputcubelist_converter(*constraints):
     These cubes get loaded and returned as a CubeList.
 
     Args:
-        *constraints (str or list):
+        *constraints (tuple of str or list):
             Constraints to be used in the loading of cubes into a CubeList.
-            If multiple constraints are specified in a list, all provided
-            constraints will be tried as possible options. Only one must
-            identify valid input, otherwise an error will be raised.
+            If the tuple contains a string then the string is expected to
+            return exactly one match. If a tuple contains a list then the list
+            is treated as a group that is expected to return a single match.
+            The tuple can contain a mixture of strings and lists, as required.
```

Comment:
```suggestion
            If the tuple contains a string or multiple strings, then each
            string is expected to return exactly one match. If the tuple
            contains a list or multiple lists, then each list is treated as a
            group that is expected to return a single match. The tuple
            can contain a mixture of strings and lists, as required.
```

Reply:
How about that, just to cover the fact that multiples of both strings and lists can be provided within this tuple?